README.md

```diff
@@ -115,7 +115,9 @@ public void ConfigureServices(IServiceCollection services)
             // For demo purposes only. Private key should be stored securely
             options.Payments.SigningKey.PrivateKey = File.ReadAllText("ec512-private-key.pem");
         }
-    });
+    })
+    // We advice to cache the auth token
```

Comment:
Do you want to do suggest the same on [Java's readme?](https://github.com/TrueLayer/truelayer-java?tab=readme-ov-file#initialize-truelayerclient) 😄 
I think we could do that in lieu of the environment and/or logs opt-in

Reply:
yeah I think it's a good advice